demo/oak-1-theora/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM ros:iron-ros-core
 
 RUN apt update && apt install -y \
-        ros-${ROS_DISTRO}-rqt*
\ No newline at end of file
+        ros-${ROS_DISTRO}-rqt*
```

Comment:
I think it is not a good idea to install all `rqt` applications or plugins. Take a look:
https://index.ros.org/search/?term=rqt
![image](https://github.com/husarion/depthai-docker/assets/109142865/93d6df89-d5f5-4703-93db-fa22d66cb23d)

Let's find what plugin is needed maybe only `rqt_image_view`?